src/MessagingAttachment/MessagingAttachment.tsx

```diff
@@ -11,8 +11,17 @@ function cssClass(element: string) {
   return `MessagingAttachment--${element}`;
 }
 
-type Props = {
+type AttachmentPreviewProps = {
+  // TODO: should any of these be optional vs required (different then they currently are)?
```

Comment:
Assuming they all still have default values matching our English copy (`attachmentURL` excluded, of course), I think this is fine!